StatePrinter.Tests/IntegrationTests/DictionaryTest.cs

```diff
@@ -76,21 +76,41 @@ public void Setup()
             public void Dictionary_empty_int_int_with_rootName()
             {
                 var expected = @"new List<Token>()
-[0] = new Token()
 {
-    Tokenkind = StartEnumeration
-    FieldType = null
-    Field = null
-    Value = null
-    ReferenceNo = null
-}
-[1] = new Token()
-{
-    Tokenkind = EndEnumeration
-    FieldType = null
-    Field = null
-    Value = null
-    ReferenceNo = null
+    new Token()
+    {
+        Tokenkind = FieldnameWithTypeAndReference
+        FieldType = new RuntimeType()
```

Comment:
That's what `object.GetType()` returns. You can see several `RuntimeType()` instances in the current code of this file (prior to my changes).